whisper_online.py

```diff
@@ -101,8 +106,9 @@ class FasterWhisperASR(ASRBase):
 
     def load_model(self, modelsize=None, cache_dir=None, model_dir=None):
         from faster_whisper import WhisperModel
+        logging.getLogger("faster_whisper").setLevel(logger.level)
```

Comment:
I'm following the `faster-whisper` docs here: https://github.com/SYSTRAN/faster-whisper/blob/91c8307aa6b305dcaf12fc112b0187ee0d512b00/README.md?plain=1#L218 - this is the intended pattern for controlling how verbose it is.

> What if the user wants to set a different log level themselves? 

That's exactly what this is for, but I can see we've got two conflicting goals here: on the one hand someone who's using `whisper_online.py` as a module might want different log levels for `faster-whisper` to everything else; on the other hand someone who's using it as a `__main__` has nowhere else to configure it at all.

Would I be right in thinking that a user can change the log level themselves at any point after this anyway?  If they can call `setLevel` themselves then this isn't constraining them, it's just propagating a default.

Reply:
True, it's used both as a module and a script. I guess I would say that the faster_whisper log level should probably set after basicConfig down below instead.

Reply:
I went for it here because the user might not be using `faster-whisper` at all.  Do it down at the `basicConfig` call and there's an additional conditional on the backend choice that we can otherwise avoid. `faster-whisper` isn't actually loaded until the import on line 108 so the logger we're configuring doesn't exist until that point. I'm not sure that's as important though.

Reply:
I understand. But I think setting the logger configuration does not actually import the package. It's just administration, so perfectly fine even if you don't use (or have) faster-wisper.